otelcol/config.go

```diff
@@ -44,6 +46,11 @@ type Config struct {
 func (cfg *Config) Validate() error {
 	// Currently, there is no default receiver enabled.
 	// The configuration must specify at least one receiver to be valid.
+
+	if cfg == nil || reflect.ValueOf(cfg).Len() < 1 {
```

Comment:
`cfg` is not a Array, Chan, Map, Slice, String, or pointer to Array, so this call will panic.  I believe we'll need to check the len of each of `cfg`'s sections and if they are all 0 then return this error.

Reply:
Alright no problem then. The reason why I did not want to check the length of individual properties of the cfg is that that config may grow. That's another property may be added to cfg in the future